test/lint/lint.js

```diff
@@ -22,14 +19,34 @@ const cli = new CLIEngine({
 
 cli.addPlugin('eslint-plugin-unicorn', unicorn);
 
+// Find a way to make sure rules are loaded from codebase
```

Comment:
Hard to to this, https://github.com/eslint/eslint/issues/12425 , will find a way